src/main/java/com/google/devtools/build/lib/exec/StandaloneTestStrategy.java

```diff
@@ -586,7 +586,6 @@ private static Spawn createCoveragePostProcessingSpawn(
             .addTransitive(action.getInputs())
             .addAll(expandedCoverageDir)
             .add(action.getCollectCoverageScript())
-            .add(action.getCoverageDirectoryTreeArtifact())
```

Comment:
I don't get why this is removed. Can you explain a bit more?

Reply:
I see. `SpawnInputExpander` can only expand inputs which are existing prior to action execution. This post-processing spawn is collecting inputs generated by previous spawn which are not visible to `SpawnInputExpander` so it is always expand to nothing -- that's the reason why we manually expand tree artifact here by reading directly from metadata provider.

I think it's safe to remove this line here. but @oquenchil can you confirm since you originally wrote this?

Reply:
Yes, it looks safe to me.